caddyhttp/proxy/upstream.go

```diff
@@ -22,7 +22,7 @@ import (
 )
 
 var (
-	supportedPolicies = make(map[string]func() Policy)
+	supportedPolicies = make(map[string]func(string) Policy)
```

Comment:
This is fine, I'm rewriting this part of the proxy middleware now and will take it into account.